plugins/woocommerce/tests/e2e-pw/global-setup.js

```diff
@@ -36,7 +36,12 @@ module.exports = async ( config ) => {
 						},
 					}
 				);
-				console.log( 'Reset successful:', response.status() );
+
+				if ( response.ok() ) {
+					console.log( 'Reset successful:', response.status() );
+				} else {
+					console.error( 'ERROR! Reset failed:', response.status() );
+				}
```

Comment:
Does it make sense to abort the test run if the reset fails?  The chances that the test run will succeed or yield accurate results is pretty low.  

Your PR is to just report the correct status -- which this does, so 👍🏼  -- but something to consider for the future.